Update node version compatibility on 8.x documentation (#1120)
PrestaShop/docs: development/compile-assets.md

## Requirements

We use [Webpack](https://webpack.js.org/) to compile assets. PrestaShop is using NodeJS ([get it here](https://nodejs.org/)), NPM will take care of it all.

Here is a list of compatibility: 

| PrestaShop Versions | NodeJS Versions |
|---------------------|-----------------|
| 8.x                 | 14.x            |

> If you can't manage to compile assets, you may be using an older version of PrestaShop. We strongly recommend you ton install a package such as ['n'](https://www.npmjs.com/package/n?activeTab=versions) or [nvm](https://github.com/nvm-sh/nvm) to try with an older version of NodeJS.

Reviewer: ```suggestion
If you can not manage to compile assets, you may be using an older version of PrestaShop. We strongly recommend you to install a package such as ['n'](https://www.npmjs.com/package/n?activeTab=versions) or [nvm](https://github.com/nvm-sh/nvm) to try with an older version of NodeJS.
```

> On Windows, you'll probably need to install `windows-build-tools` using `npm i --global windows-build-tools` in order to make [node-gyp](https://github.com/nodejs/node-gyp#on-windows) working.

Reviewer: ```suggestion
On Windows, you will probably need to install `windows-build-tools` using `npm i --global windows-build-tools` in order to make [node-gyp](https://github.com/nodejs/node-gyp#on-windows) working.
```
Maybe a little bit more formal 😉  ?

Author: What do you mean @matks ?

Reviewer: Using "you'll" or "we'll" or "what's" is a bit friendly, a bit chill 😄 the kind of tone you use with friends. Maybe for the docs we should aim for a tone a little bit more boring, more formal, more business-oriented

This is why I suggest to use `you will` instead of `you'll`
